content/4_Gold/SP.mdx

```diff
@@ -358,13 +358,11 @@ public class test {
 
 <Warning>
 
-It's important to include `continue`. This ensures that when all edge weights are non-negative, we will never go through the adjacency list of any vertex more than once. Removing it will cause TLE on the last test of the above problem!
+It's important to include `continue`. This ensures that when all edge weights are non-negative, we will never go through the adjacency list of any vertex more than once. Removing it will cause TLE on the last test of the above problem! 
 
-<IncompleteSection>
+The last testcase contains 100000 destinations and 149997 routes. City 1 has around 50000 routes to cities 2 through 50000. Cities 2 through 50000 have routes that go to city 50001. City 50001 has routes that go to cities 50002 through 100000. Without the `continue`, after the program processes the routes from city 1, the priority queue will contain 49998 routes that point to city 50001. After getting the shortest route to city 50001 from these routes, it will move on to the cities 50002 through 100000 that can be reached from City 50001. However, the priority queue now has 49998 instances of these routes. This means that city 50001 will be processed 49998 more times and so on for the 49998 cities that branches out from city 50001. This results in a worse runtime than $\mathcal{O}(N^2)$, which will TLE. 
```

Comment:
Shouldn't 49998 be 49999?

Reply:
> after the program processes the routes from city 1

shouldn't this be - after the program pops cities 1 - 50000 off the queue?

Reply:
> However, the priority queue now has 49998 instances of these routes.

which routes?

Reply:
> cities that branches out

huh

Reply:
> This results in a worse runtime than $\mathcal{O}(N^2)$, which will TLE. 

I guess you should say $\Theta(N^2\log N)$ instead.

Reply:
> City 1 has around 50000 routes to cities 2 through 50000

Why not "City 1 has flights to cities 2 through 50000"? Shouldn't use "route" and "flight" interchangeably.